Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -177,6 +182,8 @@ class FontDescription {
 
     FontSelectionRequest m_fontSelectionRequest;
     Markable<float, FloatMarkableTraits> m_sizeAdjust; // Size adjust for font-size-adjust
+    TextSpacingTrim m_textSpacingTrim;
+    TextAutospace m_textAutospace;
```

Comment:
These should go to the bitfield to avoid bloating the type.

Reply:
The longhands should support future '||' syntax with too many combinations for representing it as enum in a readable way:

`text-autospace: normal | auto | no-autospace | [ [ ideograph-alpha || ideograph-numeric || punctuation ] || [ insert | replace ] ]"`

Reply:
Not sure I see the relation to my comment.

Reply:
I mean you can put an OptionSet to a bitfield too.